routingpy/client_default.py

```diff
@@ -184,7 +106,7 @@ def _request(
         first_request_time=None,
         retry_counter=0,
         requests_kwargs=None,
```

Comment:
I just realized that this is never used in any router method (i.e. no method accepts `requests_kwargs` as arg), so this is actually superfluous. Instead of removing it though, can you put it at the end, call it `**client_kwargs`

Reply:
Actually, it does! See line 193, it's just renamed as `final_requests_kwargs`

Reply:
jep but no router actually makes use of it. i.e. if you go into valhalla.directions() there's no `request_kwargs`

Reply:
What about the `__init__` of `ORS`? There, the API key is currently passed as part of the `requests_args`.